lib/resources/packages.rb

```diff
@@ -44,31 +48,29 @@ def to_s
 
     private
 
+    def packages_command
+      os = inspec.os
+      if os.debian?
+        command = "dpkg-query -W -f='${db:Status-Abbrev} ${Package} ${Version}\\n'"
+      end
+      command
```

Comment:
I've done it in `initialize`:
`return skip_resource "The packages resource is not yet supported on OS #{inspec.os.name}" unless @command`